lib/common.sh

```diff
@@ -288,17 +288,7 @@ export FAKE_IPA_IMAGE="${FAKE_IPA_IMAGE:-${CONTAINER_REGISTRY}/metal3-io/fake-ip
 export FKAS_IMAGE="${FKAS_IMAGE:-${CONTAINER_REGISTRY}/metal3-io/metal3-fkas}"
 
 # CAPM3 and IPAM controller images
-if [[ "${CAPM3RELEASEBRANCH}" = "release-1.7" ]]; then
```

Comment:
This would also need removal of the CI jobs etc. I'd prefer doing the cleanup in separate PR.